.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Changes
+
+<!-- Describe your changes here- ideally you can get that description straight from
+your descriptive commit message(s)! -->
+
+# Submitter Checklist
+
+These are the criteria that every PR should meet, please check them off as you
+review them:
+
+TODO: add submitter checklist details
```

Comment:
probably you can copy from current catalog repository

Reply:
One thing here is that we can be way more specific than on `tektoncd/catalog`. We know it's only about `golang`, etc.. so we could get the initial checklist from the catalog repository but we may want to adapt.

Reply:
Yeah, I will use the content in the current catalog repository as a starting point. And there are some improvement and link updates that we need to do (like the golang specific checks, update the e2e test link once the contributing doc is ready...) so that's why I leave it as todo for now. Will send out PR to fill in the content later if you don't mind 😁